bayesian/utils.py

```diff
@@ -329,8 +329,118 @@ def search_packages_from_graph(tokens):
     return {'result': pkg_list}
 
 
+class GraphAnalyses:
+    """It Queries GraphDB Based on vendor and returns json converted data."""
+
+    def __init__(self, ecosystem, package, version, vendor=None):
+        """For Flows related to Security Vendor Integrations."""
+        self.vendor = vendor
+        self.ecosystem = ecosystem
+        self.version = version
+        self.package = package
+
+    @staticmethod
+    def build_graphdb_query(vendor):
+        """Func returns GraphDB query based on vendor argument."""
+        get_cve_info = """
+            g.V().has('pecosystem', ecosystem).has('pname', name).has('version', version)
+            .as('version').in('has_version').dedup().as('package').select('version')
+            .coalesce(out('has_cve').as('cve').select('package','version','cve')
+            .by(valueMap()),select('package','version').by(valueMap()));
+            """
+
+        get_latest_non_cve_version = """
```

Comment:
We dont need this query in the vendor = snyk case. Only the prev query is sufficient